docs/_blog/2022-04-12-kuma-1-6-0.md

```diff
@@ -0,0 +1,95 @@
+tags:
+  - Release
+canonicalUrl: 'https://kuma.io/blog/kuma-1-6-0-released'
+---
+
+We are happy to announce Kuma's latest release, which is packed with features and improvements, including substantial performance improvements to our built gateway and .
+
+We strongly suggest upgrading, in order to take advantage of the latest and greatest when it comes to service mesh.
+
+## Notable Features
+
+* 🚀 We provide a preview of [Kubernetes Gateway API](https://gateway-api.sigs.k8s.io/) support for our builtin gateway. This makes it easier than to provide a gateway to lead traffic through your mesh
+* 🚀 Full support for the "inspect API" on builtin gateway resources. This enables users to see which policies impact which `MeshGatewayRoute`.
+* 🚀 ZoneEgress received many improvements like: support for Standalone, locality aware routing on external services and support for `FaultInjection` and `RateLimit` policies on external services.
+* 🚀 A preview of the completely rewritten transparent-proxy, this aims to make transparent proxy more stable and provide us with pathways for further innovation.
+* Many improvements to the Helm charts like: exposing the CP with an ingress, provide resource limits to components, image tag customization.
+* A new metric to see how long configuration change take to propagate to data plane proxies.
+
+And a lot more! The full changelog is available [here](https://github.com/kumahq/kuma/blob/master/CHANGELOG.md).
+
+## Kubernetes Gateway API (experimental)
+
+The [gateway API kubernetes sig](https://gateway-api.sigs.k8s.io/) is gaining traction and quickly converging to be a flexible successor for Ingress in Kubernetes.
+Since Kuma 1.5.0 we support a builtin Envoy Gateway fully configurable with Kuma policies.
+It made sense for us to make the Kubernetes story complete by also providing Gateway API support.
+
+Gateway API is currently experimental and is enabled with the kuma-cp flag `--experimental-gatewayapi` and will also need the Gateway to be enabled with `--experimental-meshgateway`.
+
+Check the [Gateway API docs](https://kuma.io/docs/1.6.x/explore/gateway-api) for more information.
+
+## Transparent-proxy rewrite (experimental)
+
+We have always been very limited by our current transparent proxy. The code-base is mainly inherited and hard to maintain.
+We have big plans on transparent-proxy including some features using cutting edge network and kernel technologies.
+As a first step in this effort we've completely rewritten the iptables transparent-proxy rules.
+
+This is now ready and can be enabled with `kumactl install transparent-proxy --experimental-transparent-proxy-engine`.
+Please take it for a run in dev (this is not ready for production yet) and let us know of any [issue](https://github.com/kumahq/kuma/issues/new)!
+
+## ZoneEgress improvements
+
+In Kuma 1.5.0 ZoneEgress was released, it enables routing all traffic external to the mesh through a dedicated set of proxies.
```

Comment:
```suggestion
In Kuma 1.5.0 zone egress was released, it enables routing all traffic external to the mesh through a dedicated set of proxies.
```

my preference at least. Feels jarring to refer to a whole feature by the resource (and otherwise should be `ZoneEgress`), but may just be me and I don't know what other projects do.